resources/migrations/001_update_migrations.yaml

```diff
@@ -8191,6 +8191,50 @@ databaseChangeLog:
               column:
                 name: dashboard_id
 
+  # last_used_at is used to determine if a report card is stale and should be cleaned up
+  # if the card is imported with serialization, we don't want to automatically clean up
+  # newly imported cards, and we don't want to have to make it nullable.
+  - changeSet:
+      id: v50.2024-07-09T20:04:01
+      author: calherries
+      comment: Populate default value for report_card.last_used_at
+      changes:
+        - sql:
+            sql: >-
+              UPDATE report_card
+              SET last_used_at = current_timestamp
+              WHERE last_used_at IS NULL;
+      rollback: # nothing to do, since this is backwards compatible
+
+  - changeSet:
+      id: v50.2024-07-09T20:04:02
+      author: calherries
+      comment: Add not null constraint for report_card.last_used_at
+      preConditions:
+      changes:
+        - addNotNullConstraint:
+            tableName: report_card
+            columnName: last_used_at
+            columnDataType: ${timestamp_type}
+
+  # addDefaultValue with defaultValueComputed doesn't work for MySQL/MariaDB so we have to do this the hard way.
```

Comment:
I shamelessly copied the MySQL/MariaDB SQL from previous migrations, e.g. 
https://github.com/metabase/metabase/blob/706540413848f442059252662d483915551abd9c/resources/migrations/001_update_migrations.yaml#L674-L690